source/frontend/src/features/leases/models.ts

```diff
@@ -160,74 +166,43 @@ export class LeaseDetails {
 
 export class FormLeaseProperty {
   id?: number;
-  property?: FormProperty;
+  property?: PropertyForm;
   leaseId?: number;
   rowVersion?: number;
-  landArea?: number;
-  areaUnitType?: Api_TypeCode<string>;
+  landArea: string;
```

Comment:
is landArea required? or is this taking into account formik's restriction that undefined is not allowed.

Reply:
It does not look like is required on the mockups

Reply:
so... maybe should be nullable then?